tests/unit/base/test_announcement_client.py

```diff
@@ -75,7 +75,7 @@ def test_get_posts_success(self) -> None:
                 {
                     'title': 'Test Title',
                     'date': 'December 31, 1999',
-                    'info_list': ['Test announcement'],
+                    'html_content': 'content',
```

Comment:
this is a typo. There is no `info_list` anywhere. I think it passed before because even the old implementation of `dump` did not validate objects.